<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestTreeViewClass</class>
 <widget class="QDialog" name="TestTreeViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestTreeView</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,1">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_expand_all">
       <property name="text">
        <string>Expand All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_collapse_all">
       <property name="text">
        <string>Collapse All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TestTreeView.qrc"/>
 </resources>
 <connections/>
</ui>
